prestoadmin/server.py

```diff
@@ -125,6 +127,15 @@ def uninstall():
         print('Package uninstalled successfully on: ' + env.host)
 
 
+@task
+@requires_topology
+def upgrade(local_package_path, local_config_dir):
```

Comment:
I've added the pydoc, please have a look and make sure you agree with the wording.